test/e2e/storage/csi_volumes.go

```diff
@@ -78,13 +78,44 @@ var _ = Describe("PMEM Volumes", func() {
 	// List of testSuites to be executed in below loop
 	var csiTestSuites = []func() testsuites.TestSuite{
 		// TODO: investigate how useful these tests are and enable them.
+
 		// testsuites.InitMultiVolumeTestSuite,
+		// === above line refers to tests that are not found, proposing removal line from here
+
 		testsuites.InitProvisioningTestSuite,
+
 		// testsuites.InitSnapshottableTestSuite,
+		// === above line adds one skipped test (as snapshot-related get skipped), proposing removal line from here
+
 		// testsuites.InitSubPathTestSuite,
+		// === above line changes from "Ran 39 of  83 Specs in 89.985  sec, 39 Passed | 0 Failed | 0 Pending | 44 Skipped"
+		// ===                    to   "Ran 57 of 137 Specs in 925.305 sec, 55 Passed | 2 Failed | 0 Pending | 80 Skipped"
+		// === errors:
+		// === [It] should unmount if pod is force deleted while kubelet is down [Disruptive][Slow]
+		//     No external address for pod pod-subpath-test-pmem-csi-dynamicpv-k559 on node pmem-csi-clear-govm-worker2
+		//
+		// === [It] should unmount if pod is gracefully deleted while kubelet is down [Disruptive][Slow]
+		//     No external address for pod pod-subpath-test-pmem-csi-dynamicpv-q9sj on node pmem-csi-clear-govm-worker3
+		// === proposing checking/fixing errors if possible and enabling, (but adds long time to testing)
+
```

Comment:
Let's keep this disabled. Those tests are occasionally flaky also for Kubernetes, and I think they mostly test Kubernetes features, not so much the driver.